### PR TITLE
Add Patternfly 4 react charts

### PR DESCRIPTION
**Pull request by sjha4**

### CODE REVIEW

**Comment by sjha4** [comment]
Wondering why lint would fail at https://github.com/theforeman/foreman/blob/e49da5764920b3810e1c33ea2517ec8c870362b8/webpack/assets/javascripts/services/charts/LineChartService.js#L32 console statement on this PR?

File: packages/vendor-core/package.json [hunk 1]
```diff
@@ -28,9 +28,10 @@
     "@patternfly/react-catalog-view-extension": "^4.10.2",
     "@patternfly/react-core": "~4.97.2",
     "@patternfly/react-icons": "~4.9.2",
-    "@patternfly/react-styles": "~4.8.2",
+    "@patternfly/react-styles": "^4.10.2",
```
Comment (MariaAga) [inline packages/vendor-core/package.json:31]: Is this update necessary?  We want have all pf4 packages from the same release. currently we are using version 2021.02 (2021-02-17) 
https://www.patternfly.org/2021.02/developer-resources/release-notes

File: packages/vendor-core/package.json [hunk 2]
```diff
@@ -28,9 +28,10 @@
...
     "@patternfly/react-table": "~4.23.2",
-    "@patternfly/react-tokens": "~4.10.2",
+    "@patternfly/react-charts": "~6.14.17",
+    "@patternfly/react-tokens": "^4.11.3",
```
Comment (MariaAga) [inline packages/vendor-core/package.json:34]: Same for these, the matching version for react-charts is `@patternfly/react-charts@6.14.2`

**Review by MariaAga — CHANGES_REQUESTED** [review]

**Comment by MariaAga** [comment]
Test failure is unrelated

File: packages/vendor-core/package.json (hunk 2 above)
Comment (sjha4) [reply packages/vendor-core/package.json]: Updated..

**Comment by MariaAga** [comment]
Thanks @sjha4!

**Pull request merged**